runtime/__init__.py

```diff
@@ -7,7 +7,8 @@
 """Interface to the native C++ ExecuTorch runtime.
 
 Example usage:
-.. code-block:: text
+
+.. code-block:: python
```

Comment:
Just noticed that this example probably won't work as-is since it doesn't import Program or Method. Please try pasting it into a python interpreter to make sure it works